src/index.html

```diff
@@ -11,7 +11,127 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"> 
+
+      <fieldset class="bloc">
+        <legend>Personal information</legend> 
+        <label class="form-field">
+          Surname
+          <input 
+            type="text" 
+            name="surname" 
+            autocomplete="off"
+          >
+        </label> 
+        <label class="form-field">
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            autocapitalize="off"
+          >
+        </label> 
+        <label class="form-field">
+          How old are You?
+          <input 
+            type="number" 
+            name="age" 
+            min="1" 
+            max="100" 
+            value="12"
+          >
+        </label> 
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="dateOfBirth">
+        </label> 
+        <label>
+            I accept the term of the agreement
+          <input type="checkbox" name="consent">
+        </label>
+      </fieldset>
+
+      <fieldset class="bloc">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            placeholder="email@example.com" 
+            name="email" 
+            required
+          >
+        </label> 
+        <label>
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="6" 
+            maxlength="12" 
+            required
+          >
+        </label> 
+      </fieldset>
+
+      <fieldset class="bloc">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
```

Comment:
Don't use this common label, because when you click on the question, yes becomes selected
```suggestion
        <div class="form-field">
```